Citizens/templates.yml

```diff
@@ -1,3 +1,35 @@
+reverseflashdanielwest: 
+  override: false
+  replacements: 
+    metadata: 
```

Comment:
This metadata is missing the player-skin-use-latest node

player-skin-use-latest: false